commit ed812cac5ec1d76915c4b2a3049ff03735d26ce0
Author: Phil Elwell <phil@raspberrypi.org>
Date:   Mon Oct 29 10:38:31 2018 +0000

    overlays: pi3-disable-bt: Clear out bt_pins node
    
    The pi3-disable-bt overlay does not (and cannot) delete the bt_pins
    node, but emptying its properties (including brcm,pins) is a way of
    signalling to the hciuart systemd service that Bluetooth has been
    disabled.
    
    Signed-off-by: Phil Elwell <phil@raspberrypi.org>

diff --git a/arch/arm/boot/dts/overlays/pi3-disable-bt-overlay.dts b/arch/arm/boot/dts/overlays/pi3-disable-bt-overlay.dts
index 87cf345f9641..975b5ee044bc 100644
--- a/arch/arm/boot/dts/overlays/pi3-disable-bt-overlay.dts
+++ b/arch/arm/boot/dts/overlays/pi3-disable-bt-overlay.dts
@@ -37,6 +37,15 @@
 	};
 
 	fragment@3 {
+		target = <&bt_pins>;
+		__overlay__ {
+			brcm,pins;
+			brcm,function;
+			brcm,pull;
+		};
+	};
+
+	fragment@4 {
 		target-path = "/aliases";
 		__overlay__ {
 			serial0 = "/soc/serial@7e201000";
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1283_ed812cac5ec1d76915c4b2a3049ff03735d26ce0.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1283_ed812cac5ec1d76915c4b2a3049ff03735d26ce0.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
